<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoD - Papers Past Reporting Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="directory_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Directory</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="directory_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="start_date_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start date (YYYYMMDD)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="start_date_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="end_date_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="end_date_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>End date (YYYYMMDD)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="results_box">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>491</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_code_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title_code_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Title code </string>
    </property>
   </widget>
   <widget class="QPushButton" name="directory_button">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="progress_text">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>491</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="issue_group">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Count Issues</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="size_check">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Get total size</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="file_group">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Count File Types</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <widget class="QGroupBox" name="ie_mets_group">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>IE_METS</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>IE_METS</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="pm_group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>PM_01</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>67</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TIFF</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="ac_group">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>AC_01</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="ac_page_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Page PDF</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ac_issue_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Issue PDF</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="mm_group">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="title">
      <string>MM_01</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="mm_tiff_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>TIFF</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="mm_alto_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>ALTO</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="mm_mets_check">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>METS</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>directory_input</tabstop>
  <tabstop>directory_button</tabstop>
  <tabstop>title_code_input</tabstop>
  <tabstop>start_date_input</tabstop>
  <tabstop>end_date_input</tabstop>
  <tabstop>start_button</tabstop>
  <tabstop>progress_text</tabstop>
  <tabstop>results_box</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
